### PR TITLE
Connecting Dots in C++

### PR DESCRIPTION
**Pull request by gayatri517**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Code added in cp(competitive programming) folder.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#5765

### CODE REVIEW

File: C-Plus-Plus/cp/connecting_dots.cpp [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    sort(a1,a1+white);
+    sort(a2,a2+black);
+    for(int i=0;i<white;i++)
+    {
+        int temp=abs(a1[i]-a2[i]);
+        ans+=temp;
+    }
+    return ans;
+}
+int main()
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/connecting_dots.cpp:23]: Add a new line above main

File: C-Plus-Plus/cp/connecting_dots.cpp [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    int a1[100],a2[100];
+    for(int i=0;i<white;i++)
+    {
+        cin>>a1[i];
+    }
+    for(int j=0;j<black;j++)
+    {
+        cin>>a2[j];
+    }
+    cout<<"total length needed is: "<<connectDots(a1,a2,white,black)<<endl;
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/connecting_dots.cpp]: Capitalize `T ` of total

File: C-Plus-Plus/cp/connecting_dots.cpp [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+Input:-
+Enter total number of white and black dots:
+3 3
+3 4 1
+2 5 6
+Output:-
+total length needed is: 5
+Time Complexity: O(nlogn)
+Space Complexity: O(n)
+*/
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/connecting_dots.cpp]: Rewrite it like that
```
/*
Example: 

Input:-
Enter total number of white and black dots:
3 3
3 4 1
2 5 6

Output:-
Total length needed is: 5

Time Complexity: O(nlogn)
Space Complexity: O(n)
*/
```

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by gayatri517** [comment]
@ankitaggarwal23 i have made the changes. Please review it now

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**